underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
Hows this look @jridgewell https://github.com/jashkenas/underscore/compare/master...megawac:chunk

I'm still on the fence about `<1` case. I think empty array may be best